alpha/lib/model/category.php

```diff
@@ -49,6 +56,45 @@ class category extends Basecategory implements IIndexable, IRelatedObject
 	 */
 	protected static $incrementedEntryIds = array();
 	
+	/**
+	 * @return bool
+	 */
+	public function getIsAggregationCategory() {
+		return $this->getFromCustomData(self::IS_AGGREGATION_CATEGORY, null, false);
+	}
+
+	/**
+	 * @return string
+	 */
+	public function getAggregationCategories() {
+		return $this->getFromCustomData(self::AGGREGATION_CATEGORIES, null, "");
+	}
+	
+	/**
+	 * @return string
+	 */
+	public function getPublishingCategories () {
+		return $this->getFromCustomData(self::PUBLISHING_CATEGORIES, null, "");
+	}
+
+	/**
+	 * @param bool $v
+	 */
+	public function setIsAggregationCategory($v) {
+		$this->putInCustomData(self::IS_AGGREGATION_CATEGORY, $v);
+	}
+
+	/**
+	 * @param string $value
+	 */
+	public function setAggregationCategories($value) {
```

Comment:
For `setAggregationCategories` & `setPublishingCategories` it would also make sense to pass an array instead of string.
It will make the code that is using it much cleaner and you won't need to duplicate the `implode(',', $xxx)`, `explode(',', $xxx)`